### DISCUSSION TITLE
[Support]: Frigate on Proxmox LXC stuck loading at web ui, can't edit config, can't find config from console to edit with nano

### DISCUSSION THREAD

**Discussion by bobloadmire** [blakeblackshear/frigate#11006] [General Support]
### Describe the problem you are having

Frigate stuck loading at web ui, can't edit config, can't find config from console to edit with nano. In console "ls -a" lists a bunch o .* files. How do I find and edit the config.yaml?

### Version

no idea, not accessible.

### Frigate config file

```yaml
no idea, not accessible.
```

### Relevant log output

```shell
no idea, not accessible.
```

### FFprobe output from your camera

```shell
no idea, not accessible.
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

Wyze Bridge

### Any other information that may be helpful

I'm on Proxmox using the Proxmox Helper Scripts from https://tteck.github.io/Proxmox/. It was working fine before I somehow messed up the config. I just want to access the config with nano so I can fix it. console at boot shows no logs.

**Comment by NickM-27** [comment]
only you will know where the config is stored on your system

**Reply by bobloadmire** [reply]
i don't know how to find it since ls at root lists nothing. whats going on?
